README.rst

```diff
@@ -68,15 +69,17 @@ By default, the PyAnsys package installs these core modules:
 - `PyPIM`_
 - `PyPrimeMesh`_
 - `PySeascape`_
+- `PySystemCoupling`_
 - `PyTwin`_
 - `Granta MI BoM Analytics`_
 - `Shared Components`_
 
-However, the ``pyansys`` package also contains certain extra targets, which can be installed upon request:
+Additionally, the ``pyansys`` metapackage contains certain extra targets that
+can be installed upon request:
 
-- **mapdl-all**: this target installs the core packages and `PyMAPDL Reader`_.
-- **fluent-all**: this target installs the core packages and `PyFluent-Parametric`_ and `PyFluent-Visualization`_.
-- **all**: this target install all extra ``pyansys`` packages.
+- **mapdl-all**: This target installs the core PyMAPDL package and `PyMAPDL Reader`_.
+- **fluent-all**: This target installs the core PyFluent package and `PyFluent-Parametric`_ and `PyFluent-Visualization`_.
```

Comment:
@Roberto--Why remove the "Py" library name? I can see changing to plural for the Fluent entry if both PyFluent and PyFluent Composites are both installed as core PyFluent packages. However, why plural for PyMAPDL if only it (and not PyMAPDL Reader) are installed as core?

Reply:
That's because it's not installing PyMAPDL only. It's installing all core packages AND some extra MAPDL releated packages (PyMAPDL-Reader in this case). The same happens with ``fluent-all``. We install all core packages and the extra PyFluent ones (PyFluent-Parametric and PyFluent Visualization).

Hope it's more clear now 😄